cli/tre/commands/shared_services/shared_service.py

```diff
@@ -0,0 +1,105 @@
+import click
+import logging
+
+from tre.api_client import ApiClient
+from tre.commands.operation import operation_show
+from tre.output import output, output_option, query_option
+from .contexts import pass_shared_service_context, SharedServiceContext
+
+from .operation import shared_service_operation
+from .operations import shared_service_operations
+
+
+@click.group(invoke_without_command=True, help="Perform actions on an individual shared_service")
+@click.argument('shared_service_id', required=True, type=click.UUID)
+@click.pass_context
+def shared_service(ctx: click.Context, shared_service_id: str) -> None:
+    ctx.obj = SharedServiceContext(shared_service_id)
+
+
+@click.command(name="show", help="Show a shared_service")
+@output_option()
+@query_option()
+@pass_shared_service_context
+def shared_service_show(shared_service_context: SharedServiceContext, output_format, query):
+    log = logging.getLogger(__name__)
+
+    shared_service_id = shared_service_context.shared_service_id
+    if shared_service_id is None:
+        raise click.UsageError('Missing shared_service ID')
+
+    client = ApiClient.get_api_client_from_config()
+    response = client.call_api(log, 'GET', f'/api/shared-services/{shared_service_id}', )
+    output(response.text, output_format=output_format, query=query, default_table_query=r"sharedServices[].{id:id,name:templateName, version:templateVersion, is_enabled:isEnabled, status: deploymentStatus}")
+
+#
+# TODO - add PATCH (and ?set-enabled)
```

Comment:
do we need these TODOs in here and, if so, should we group this one with the one below?

Reply:
Grouped with the one below as suggested